contracts/test/SimpleNumberStorage.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.8;
+
```

Comment:
Good idea to have test contracts - just add to .npmignore so they don't become part of the build

Reply:
added